src/pages/setting/preference.tsx

```diff
@@ -159,6 +162,16 @@ export default function Preference() {
         Zap Amount
       </List.Item>
 
+      
+      <List.Item
+        actions={[
+          
```

Comment:
```suggestion
          key={'btc-connect'}
```
this should fixed the compiled error